### ISSUE TITLE
Support SSR

### ISSUE THREAD

**Issue by cco3** [shenoy/vue-scrollama#9]
Importing vue-scrollama gives `window not defined` when in SSR.

**Comment by vgshenoy** [comment]
Setting up scroll interactions is inherently client-side so for now you'll have to render the <Scrollama> component client-side only.
https://nuxtjs.org/api/components-client-only/
https://github.com/egoist/vue-client-only

**Comment by cco3** [comment]
Right, but typically tools like this make it so that you can import them in SSR even if they only render on the client.  The problem with just using a client-only tag in gridsome (which is what I'm using) is that it means my content inside of my scrollama tag loads after the rest of the page.

**Comment by ilicmarko** [comment]
@cco3 One possible solution is to load it only on client side. In your main.js file.

```js
if (process.isClient) {
    const VueScroll = require('vue-scrollama').default;
    
    Vue.use(VueScroll);
}
```

**Comment by cco3** [comment]
Right, I can do that, but then I still have to hide the component behind the client-only tag, which leads to the same issue of being loaded after everything else.

What would be nice is if the scrollama component did that test before loading the underlying library and applying it to the element.  That way the element could be rendered on both the client and the server, but only on the client would the scrollama magic get applied.

**Comment by nazar-lynovetsky** [comment]
@vgshenoy Hello!
You are planning to implement what @cco3 was talking about?
I am working with Nuxt and faced exactly the same problem when using vue-scrollama library.

**Comment by vgshenoy** [comment]
Hi folks, finally got down to this 🙈 . I've published 1.2.0 which works with SSR as per my tests. If there's still an issue, I'd be grateful if you'd let me know. Thanks!

**Comment by cco3** [comment]
Finally got around to trying it.  It works!  Thanks!
